### PR TITLE
[webui] Improve accept/decline buttons styling

### LINKED ISSUE #2327: Improve Accept/Decline buttons
Moved from https://github.com/openSUSE/obs-build/issues/306 .

It sometimes happens that we reviewers do not position our mice accurately enough and hit the wrong button in the review view (e.g. https://build.opensuse.org/request/show/440116 ).

Please consider adding some spacing between the "Approve" and "Disregard" buttons. 4em seems like a reasonable value; alternatively, right-aligning the Disregard button might also work.

Cc @DimStar77 
rt

@hennevogel also suggested to colorize the buttons, which I would also agree on (but rather for looks than effect), but would have made a separate ticket for. (But maybe it's better to do it all in one?)

### PR DESCRIPTION
**Pull request by DavidKang**
Fixes #2327

### CODE REVIEW

**Comment by hennevogel** [comment]
@DavidKang when we are proposing visual changes we include an before/after screenshot. please do so :-)

**Comment by DavidKang** [comment]
[Improve Accept/Decline buttons](https://trello.com/c/igSO40eE/134-improve-accept-decline-buttons)

**Comment by DavidKang** [comment]
Before:
screenshot before

After:
screenshot after

**Comment by bgeuken** [comment]
To me the gap is to big now. Maybe rather something like 2 to 3 chars? Not sure how much width in em that would be;-)

**Comment by coolo** [comment]
yes, 4em is heavy. This is just about a review, not about firing missles or data loss. Also the colours are heavy. I can't see this as improvement at all

**Comment by Ana06** [comment]
I also think that the colors are too jazzy and more dificult to read (specially the red one). In case that we want to keep the colors I would change the letter color to white to improve readability, although maybe this way is even jazzier. :confused:

**Comment by bgeuken** [comment]
We could also just apply a decent "ring" of green / red around the inner part of the button. More like a shadow. I wish I could paint ....

Or we keep it gray:-)

**Comment by Ana06** [comment]
@bgeuken 

> We could also just apply a decent "ring" of green / red around the inner part of the button. More like a shadow. 

I think that would be strange, as we don't have any similar in OBS (at least I haven't seen it :joy:)

**Comment by DavidKang** [comment]
screenshot after

With grey buttons :)

**Comment by bgeuken** [comment]
cool

File: src/api/app/assets/stylesheets/webui/application/request.scss [hunk 1]
```diff
@@ -8,4 +8,14 @@
 }
 .request-critical {
   color: #d80000;
-}
\ No newline at end of file
+}
+
+.review_descision_display {
```
Comment (Ana06) [inline src/api/app/assets/stylesheets/webui/application/request.scss]: Top late but why not just this:

``` css
[id^='review_accept_button']{
  margin-right: 1em;
}
```

I don't like to set the `margin-right` for all inputs with submit type inside `review_decision_display` and then remove it for the `Disregards` button using that it is the last one. In my opinion it is better to directly set the margin to the `Approve` button.

@DavidKang @bgeuken what do you think?

**Pull request merged**